mediafile.py

```diff
@@ -1801,8 +1813,17 @@ def update(self, dict):
         MP3DescStorageStyle(u'BARCODE'),
         MP4StorageStyle('----:com.apple.iTunes:BARCODE'),
         StorageStyle('BARCODE'),
+        StorageStyle('UPC', read_only=True),
```

Comment:
Effect I wanted to achieve was reading data from this 3 tags and saving only BARCODE tag.
Is it a way to do this?

Reply:
Interesting! Can you explain a little more about why you want that behavior? (I think this is the way to achieve it, but some testing would be useful, probably.)

Reply:
UPC, UPN and EAN are some specific tags for barcodes which are not supported on most of formats and by most of players.
It would be cool to fill main tag with data from those.

Reply:
Got it... but I guess what I was asking about was a little more detail on which software *does* support those fields. It seems somewhat harmless to write fields that are unpopular, if that has the effect of increasing compatibility. (That's the usual policy we use in this library.)

Reply:
Problem is that you can't easly guess if it's UPC, UPN or EAN, all of them are barcodes but those are different standards for those. UPC and UPN is the same, but EAN is other standard, all of them are BARCODEs tho and should fill main BARCODE tag.

Reply:
UPC + UPN: https://en.wikipedia.org/wiki/Universal_Product_Code
EAN: https://en.wikipedia.org/wiki/International_Article_Number